prow/pod-utils/decorate/podspec.go

```diff
@@ -23,36 +23,204 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"k8s.io/test-infra/prow/clonerefs"
+	"k8s.io/test-infra/prow/entrypoint"
+	"k8s.io/test-infra/prow/gcsupload"
+	"k8s.io/test-infra/prow/initupload"
 	"k8s.io/test-infra/prow/kube"
+	"k8s.io/test-infra/prow/pod-utils/clone"
 	"k8s.io/test-infra/prow/pod-utils/downwardapi"
+	"k8s.io/test-infra/prow/pod-utils/wrapper"
+	"k8s.io/test-infra/prow/sidecar"
 )
 
+const (
+	LogMountName            = "logs"
+	LogMountPath            = "/logs"
+	CodeMountName           = "code"
+	CodeMountPath           = "/go"
```

Comment:
Is this ok for non-golang projects?

Reply:
Figuring out what this should be is one of the TODOs in the body, /cc @BenTheElder

Reply:
/data ? /src ? /code ?

Reply:
this should be configurable?